src/client/css/partials/floatingBanner.css

```diff
@@ -0,0 +1,50 @@
+.floating-banner {
```

Comment:
Does this file need a copyright header? Or are all these css files concatenated?
Although I did notice we have one on src/client/css/partials/landing.css partial below.

```
/* This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 *
```

Reply:
Thanks for catching this one — just missed adding it.

Reply:
And because I can't resist an overly complex `grep`-ertunity, here's a few other places that might be suspiciously missing copyrights. Although this is probably best discussed in a new ticket (or not at all):

```sh
git grep -L "This Source Code Form is subject to the terms of the Mozilla Public" ':src/**' ':!*.svg' ':!*.webp' ':!*.png' ':!*.woff2' ':!*.ico' ':!*.sh' | cat

src/app/(proper_react)/components/ControlsWrapper.module.scss
src/app/(proper_react)/layout.module.scss
src/app/components/server/Icons.module.scss
src/app/globals.css
src/app/not-found.module.scss
src/app/tokens.scss
src/client/css/partials/addEmail.css
src/client/css/partials/allBreaches.css
src/client/css/partials/breachDetail.css
src/client/css/partials/error.css
src/client/css/partials/exposureScan.css
src/client/css/partials/notFound.css
src/client/css/partials/settings.css
src/client/css/userMenu.css
src/db/tables/subscribers_types.d.ts
```